kartograf/context.py

```diff
@@ -44,47 +44,52 @@ def __init__(self, args):
             self.epoch_dir = str(int(utc_time_now))
             self.epoch_datetime = datetime.utcfromtimestamp(int(utc_time_now))
 
-        cwd = os.getcwd()
+        cwd = Path.cwd()
         # Data dir
         if self.reproduce:
             if not self.args.reproduce.endswith('/'):
                 self.args.reproduce += '/'
             self.data_dir = self.args.reproduce
         else:
-            self.data_dir = f"{cwd}/data/{self.epoch_dir}/"
-        self.data_dir_irr = f"{self.data_dir}irr/"
-        self.data_dir_rpki_cache = f"{self.data_dir}rpki/cache/"
-        self.data_dir_rpki_tals = f"{self.data_dir}rpki/tals/"
-        self.data_dir_collectors = f"{self.data_dir}collectors/"
+            self.data_dir = str(cwd / "data" / self.epoch_dir)
+
+        if Path(self.data_dir).exists() and not self.reproduce:
+            print("Not so fast, a folder with that epoch already exists.")
+            sys.exit()
+
+        self.data_dir_irr = str(Path(self.data_dir) / "irr")
+        self.data_dir_rpki_cache = str(Path(self.data_dir) / "rpki/cache")
+        self.data_dir_rpki_tals = str(Path(self.data_dir) / "rpki/tals")
```

Comment:
Could you split these as well?

```suggestion
        self.data_dir_rpki_cache = str(Path(self.data_dir) / "rpki" / "cache")
        self.data_dir_rpki_tals = str(Path(self.data_dir) / "rpki" / "tals")
```